action_tutorials_cpp/src/fibonacci_action_client.cpp

```diff
@@ -0,0 +1,139 @@
+// Copyright 2019 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <inttypes.h>
+#include <memory>
+#include <string>
+#include <iostream>
+#include "example_interfaces/action/fibonacci.hpp"
```

Comment:
I think it would be better if the tutorial code used `action_tutorials/action/Fibonacci.action`. This keeps them more self-contained. This is also the action type used in the Python tutorials.